aws/config.go

```diff
@@ -146,6 +146,10 @@ type Config struct {
 	// See https://docs.aws.amazon.com/sdkref/latest/guide/settings-reference.html for
 	// more information on environment variables and shared config settings.
 	AppID string
+
+	// BaseEndpoint is an intermediary transfer location to a service specific
+	// BaseEndpoint on a service's Options.
+	BaseEndpoint *string
```

Comment:
[nonblocking] Can we `string` this?

Reply:
i agree, i should have done that. but reason why i didnt is because the corresponding field in a services `Options` struct is also a `*string`. and its much easier to do the transfer from aws config to service options when they are the same types